Week1/homework/js2-exercises/1-books.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Book list</title>
+</head>
+
+<body>
+  <h1>My book list</h1>
+
+  <script>
+    const books = [
+    {
+    title : 'An Artist of the Floating World',
+    author : 'Kazuo Ishiguro',
+    alreadyRead :false  
+    },
+    {
+    title : 'Underworld',
+    author : 'Don DeLillo',
+    alreadyRead :true   
+    }
+];
+
+    const ul = document.createElement('ul');
+    // 1
+    for (let obj of books) {
+// 2
+      const p = document.createElement('p');
+      p.innerText = `${obj.title} was written by ${obj.author.toUpperCase()}`;
+// 5
+      
+      if (obj.alreadyRead === true) {
+        p.style.color = 'green';
```

Comment:
instead of using an inline style you can have a css file with class red/green (for example) and toggle classes here is an article that explains more about this: 
https://javascript.info/styles-and-classes